build.cmd

```diff
@@ -26,6 +26,7 @@ set __SkipTestBuild=
 if "%1" == "" goto ArgsDone
 if /i "%1" == "/?" goto Usage
 if /i "%1" == "x64"    (set __BuildArch=x64&&shift&goto Arg_Loop)
+if /i "%1" == "x86"    (set __BuildArch=x86&&shift&goto Arg_Loop)
```

Comment:
I tried cross-compiling CoreCLR for i386 on FreeBSD x64 with this change: https://gist.github.com/jasonwilliams200OK/355311ec3df87c01a7af and installing i386 libunwind (`setenv ABI "freebsd:10:x86:32" ; pkg install libunwind` after successfully building for [amd64 on clean minimal base VM](https://gist.github.com/jasonwilliams200OK/06db121ec534821fe541) without installing ports or jail..).
The compilation is failing with 64-bit types errors and size constants. Seems like we would need to make the code changes as well to get it running on Unix-likes.

Hopefully it will help you somehow in your 32-bitness endeavors! :smile: